shared/data/constants.js

```diff
@@ -99,13 +116,13 @@ module.exports = {
         },
         {
             name: 'tds',
-            url: 'https://staticcdn.duckduckgo.com/trackerblocking/v6/current/extension-tds.json',
+            url: getTDSEndpoint('v6/current'),
             format: 'json',
             source: 'external',
             channels: {
-                live: 'https://staticcdn.duckduckgo.com/trackerblocking/v6/current/extension-tds.json',
-                next: 'https://staticcdn.duckduckgo.com/trackerblocking/v6/next/extension-tds.json',
-                beta: 'https://staticcdn.duckduckgo.com/trackerblocking/beta/extension-tds.json'
+                live: getTDSEndpoint('v6/current'),
```

Comment:
Please add a mock version to the `integration-test/data/staticcdn/...` folder, so they can be loaded in tests. You can probably just move the v4 version to preserve that test mock/

Reply:
Done. I've removed `v4` and added the `v6/current`, `v6/next`, and `beta` files for both MV2 and MV3.